instance/factories.py

```diff
@@ -145,6 +145,7 @@ def production_instance_factory(**kwargs):
         configuration_version=settings.STABLE_CONFIGURATION_VERSION,
         openedx_release=settings.OPENEDX_RELEASE_STABLE_REF,
         configuration_extra_settings=extra_settings,
+        openstack_server_flavor=settings.OPENSTACK_PRODUCTION_FLAVOR,
```

Comment:
@lgp171188 It would be nice to add a comment above this line that explains the purpose of setting that field here. Something like:

```python
# Allow production instances to use larger instance types by default.
```